src/eq_schema/Group.js

```diff
@@ -34,8 +34,12 @@ class Group {
     );
   }
 
-  buildBlocks(pages, groupId, ctx) {
-    return pages.map(page => new Block(page, groupId, ctx));
+  buildBlocks(pages, groupId, introduction, ctx) {
+    const blocks = pages.map(page => new Block(page, groupId, ctx));
+    if (introduction.introductionEnabled) {
+      return concat(Block.buildIntroBlock(introduction, groupId), blocks);
```

Comment:
```suggestion
      return blocks.concat(Block.buildIntroBlock(introduction, groupId));
```

But we'd also need to remove the import.

Reply:
Wouldn't this put the intro blocks after the blocks array though? And you can't do it the other way as buildIntroBlock doesn't return an array?

Reply:
```suggestion
      return [Block.buildIntroBlock(introduction, groupId), ...blocks];
```

Good point. I missed that!